src/geom/boxplot.jl

```diff
@@ -1,14 +1,19 @@
 struct BoxplotGeometry <: Gadfly.GeometryElement
     default_statistic::Gadfly.StatisticElement
     suppress_outliers::Bool
+    suppress_fences::Bool
+    candlestick::Bool
     tag::Symbol
 end
 
-BoxplotGeometry(; method=:tukey, suppress_outliers=false, tag=empty_tag) =
-    BoxplotGeometry(Gadfly.Stat.boxplot(method=method), suppress_outliers, tag)
+function BoxplotGeometry(; method=:tukey, suppress_outliers=false, suppress_fences=false,
+                           candlestick=false, tag=empty_tag)
+    stat = candlestick ? Gadfly.Stat.identity() : Gadfly.Stat.boxplot(method=method)
```

Comment:
I don't like the use of `candlestick` as a `Bool` here. For a better example of the preferred style, see the Gadfly polygon.jl file, and `Geom.ellipse` .  Note there is no `ellipse::Bool` within `struct PolygonGeometry`.